creativecloud/blocks/checkmark-cta/checkmark-cta.js

```diff
@@ -0,0 +1,37 @@
+import { getLibs } from '../../scripts/utils.js';
+
+const miloLibs = getLibs();
+const { createTag } = await import(`${miloLibs}/utils/utils.js`);
+const { decorateBlockAnalytics, decorateLinkAnalytics } = await import(`${miloLibs}/martech/attributes.js`);
+
+const checkmarkCTA = (block) => {
```

Comment:
Can we please have this function name as init to follow the milo nomenclature for blocks?